spec/amp-story-player.md

```diff
@@ -0,0 +1,201 @@
+---
+$category@: presentation
+formats:
+  - websites
+  - stories
+teaser:
+  text: A player for embedding and playing your favorite stories in your own website.
+---
+
+<!--
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="`amp-story-player`"></a> `amp-story-player`
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-player example" width="300" height="533" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/spec/img/amp-story-player.gif">
+    <noscript>
+    <img alt="amp-story-player example" src="https://github.com/ampproject/spec/img/amp-story-player.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>An easy way to embed and play stories in a website.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Scripts</strong></td>
+    <td>
+    <code>&lt;script async src="https://cdn.ampproject.org/amp-story-player-v0.js">&lt;/script></code>
+    <code>&lt;link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'></code>
+    </td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><ul>
+      <li>See <a href="https://github.com/ampproject/amphtml/blob/master/examples/amp-story/player.html">code snippet</a>.</li>
+    </ul></td>
+  </tr>
+</table>
+
+[TOC]
+
+## AMP story player usage
+
+Use `amp-story-player` to embed and play stories within a webpage.
+
+### Embed in a non-AMP page
+
+The following example demonstrates a basic embed of <amp-story-player> in a non-AMP webpage.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+    <amp-story-player style="width: 360px; height: 600px;">
+      <a
+        href="https://example.com/story.html"
+        style="--story-player-poster: url('https://example.com/img/promo3x4.jpg');"
+      >
+        <span>A description for my story.</span>
+      </a>
+    </amp-story-player>
+  </body>
+</html>
+```
+
+The high-level API for the player looks like the following:
+
+[example preview="top-frame" playground="true"]
+
+```html
```

Comment:
I saw from your comment you're unable to test this - do you mean on amp.dev or the actual example? I tried this example and it didn't work for me... but I might be doing something wrong too.

Reply:
Yeah I'm not sure how to see a preview of what will eventually show up in amp.dev... How did you try the example?

Reply:
I just threw it into the amp playground and ignored the validator. lol. It actually worked this time!